pacman-nodejs-app/docs/pacman-nodejs-app-aws-migration.md

```diff
@@ -0,0 +1,532 @@
+# Pac-Man AWS Migration
+
+This tutorial walks you through the steps to perform a Pac-Man migration from an AWS cluster A to a GKE cluster B using GKE for hosting. The tutorial documents the steps
```

Comment:
I wanted to specify that google cloud dns must be aware of the ns records/ip addresses associated with both your domain name and the aws cluster. I'm not sure what the right way to specify this is.